src/connector/src/source/filesystem/file_common.rs

```diff
@@ -141,3 +142,11 @@ pub struct FsPageItem {
 }
 
 pub type FsPage = Vec<FsPageItem>;
+
+#[derive(Debug, Clone, PartialEq, Display, Deserialize)]
+pub enum DecompressionFormat {
```

Comment:
By convention, I think it is better to name this `CompressionFormat`. This also aligns with the variable name used in other places of this PR.